nodeup/pkg/model/ntp.go

```diff
@@ -46,7 +46,10 @@ func (b *NTPBuilder) Build(c *fi.ModelBuilderContext) error {
 		return nil
 	}
 
-	if b.Distribution.IsDebianFamily() {
+	if b.Distribution == distros.DistributionFocal {
```

Comment:
```suggestion
	if b.Distribution.IsUbuntu() && b.Distribution != distros.DistributionXenial && b.Distribution != distros.DistributionBionic {
```

Reply:
Would it be ok to leave it as is? Seems simpler.

Reply:
The issue would be whether a new major version of Ubuntu is used with kops 1.17. But I suppose code would have to be added to 1.17 to support it and that's not likely.

Reply:
Exactly. I don't think there is any plan to support any other distro in the near future. Thanks!